apps/examples/wc-vanilla/index.html

```diff
@@ -34,5 +34,7 @@
         <script type="module" src="/main.js"></script>
         <script type="module" src="./node_modules/@justeattakeaway/pie-button"></script>
         <script type="module" src="./node_modules/@justeattakeaway/pie-icon-button"></script>
+        <script type="module" src="./node_modules/@justeattakeaway/pie-icons-webc/icons/IconClose"></script>
```

Comment:
@jamieomaguire What do we think about trying to keep the path the same in our PR so we don't have to update all of these?

Reply:
If you can, that would be great 👍